lib/jormungandr/test/integration/Cardano/Wallet/Jormungandr/Launch.hs

```diff
@@ -44,16 +43,19 @@ withConfig = bracket setupConfig teardownConfig
 
 setupConfig :: IO JormungandrConfig
 setupConfig = do
+    block0 <- getDataFileName "jormungandr/block0.bin"
+    secret <- getDataFileName "jormungandr/secret.yaml"
+    config <- getDataFileName "jormungandr/config.yaml"
```

Comment:
Shouldn't we be using `</>` here?

Reply:
Apparently no. Unless maybe we are compiling from windows. It's quite unclear from the cabal doc but it seems that in code, this really acts as a label and `getDataFileName` resolves it to a name that is compatible with the underlying operating system:

(source: https://www.haskell.org/cabal/users-guide/developing-packages.html?highlight=data%20dir#accessing-data-files-from-package-code)
```
If the argument is a filename listed in the data-files field, the result is the name of the corresponding file on the system on which the program is running.
````